### ISSUE TITLE
Checkpoint 1 Ready for Review

### ISSUE THREAD

**Issue by tvdmitrii** [tvdmitrii/courseSchedulePlanner-api#1]
@pawaitemadisoncollege checkpoint 1 is complete.

**Comment by pawaitemadisoncollege** [comment]
Hi @tvdmitrii!

**Problem statement** 

Does a good job clearly explaining the solution.  I'm looking forward to seeing the first iteration of this! 

- [ ] Would it make sense to specify which school schedule this is for, so that somebody reading it doesn't assume it will work for all schools right now?

**User stories w/ MVP stories identified** Your users stories are clear and well-written following the best practices for user stories. They clearly communicate what your application will do. Your MVP stories seem to include what's needed for the ent java project requirements like crud, and multiple tables with more than one one-to-many relationship. I think you've done a good job scoping the project to what's doable this semester!

**Wireframes**. Good work on these! The wireframes clearly communicate what each page will do and seem to match up with the user stories above. They look professional too! 

- [ ] Something to know is that we'll be using AWS cognito for sign up/sign in capability, so there are some screens that you may not have to worry about as they can be provided by Cognito itself.  We'll learn more about this in week 7.

**Project Plan** A good start!

- [ ] I recommend at least identifying which user story you plan to work on each week.   I think the more detailed you can be the better. What do you think about creating a task for each story to decide which jsp/s, controllers, entities, tables you'll need for that story.  Revising the plan could be a task on the plan itself (woah!)

**Reflection/Time log** Keep up the good work with this. 

*********
**status for checkpoint 1: met**

**Comment by tvdmitrii** [comment]
Thank you for the feedback!

1. I did not want to box myself in just yet in case something does not workout with MATC courses and I have to come up with my own. But I will make sure to make it more specific once the time comes.
2. I knew about Cognito, so I did not put that much effort into Sign In / Sign Up. I was using [FatBikeTrailReports](https://github.com/mad-ent-java-f24/FatBikeTrailReports) as an example of what kinds of pages I should have.
3. That is a good point. My project plan does need some work, I will add user stories to it. I am hoping to have the whole Front-End -> API -> Database pipeline working either this week or next week. After that I can put more time into implementing the stories.
